crates/ra_cfg/src/lib.rs

```diff
@@ -53,4 +53,13 @@ impl CfgOptions {
     pub fn insert_features(&mut self, iter: impl IntoIterator<Item = SmolStr>) {
         iter.into_iter().for_each(|feat| self.insert_key_value("feature".into(), feat));
     }
+
+    /// Shortcut to set cfgs
+    pub fn insert_cfgs(&mut self, iter: impl IntoIterator<Item = SmolStr>) {
```

Comment:
I think this does not belong here (and the above `inseret_features` probably as well) -- `CfgOptions` should a be pretty abstract thing, exactly what `rustc` sees. 

Lowering various command-line args to this data structure should happen where we get the args.